dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops

```diff
@@ -361,3 +361,23 @@ ERROR_BUILDING_SIGNED_JWE.useraction=Check the error that is included in the mes
 KEY_MANAGEMENT_KEY_NOT_FOUND.explanation=A key management key is a key that is used to encrypt the Content Encryption Key of a JWE token. A key management key must be specified so the JWT builder can create JWE tokens. A key management key can be specified by using the key alias and configured trust store values that are specified in the message or by using the JwtBuilder API.
 KEY_MANAGEMENT_KEY_NOT_FOUND.useraction=Verify that a key with the alias that is specified in the message exists in the trust store that is specified in the message. If the JwtBuilder API is used, ensure that a key management key is provided.
+
+# Do not translate "MicroProfile Config", "mp.jwt.verify.publickey.location"
+# 0=mp.jwt.decrypt.key.location
+DECRYPT_KEY_LOCATION_INLINE_KEY=CWWKS6062E: The value for the [{0}] MicroProfile Config property appears to include a private key string, which is not a supported configuration.
+DECRYPT_KEY_LOCATION_INLINE_KEY.explanation=The MicroProfile JWT feature does not support specifying private key strings in the MicroProfile Config property that is specified in the message. The value of this property must be a relative path or URL.
+DECRYPT_KEY_LOCATION_INLINE_KEY.useraction=Use a relative path or URL as the value for the MicroProfile Config property that is specified in the message. The location values that are supported are the same as the mp.jwt.verify.publickey.location property.
+
+# Do not translate "mp.jwt.decrypt.key.location", "keyManagementKeyAlias", "Content Encryption Key"
+# 0=JWT consumer config ID
+JWS_REQUIRED_BUT_TOKEN_NOT_JWS=CWWKS6063E: The token in the request is not in JSON Web Signature (JWS) format, but only tokens that are in JWS format are accepted. If the token is in JSON Web Encryption (JWE) format, configure the mp.jwt.decrypt.key.location MicroProfile Config property or the keyManagementKeyAlias attribute in the [{0}] JWT consumer configuration.
+JWS_REQUIRED_BUT_TOKEN_NOT_JWS.explanation=The MicroProfile JWT specification requires that only tokens in JWS format must be accepted if the JWE decryption key location is not configured. The decryption key location is missing from the MicroProfile Config properties and the server configuration, so only tokens in JWS format are accepted.
+JWS_REQUIRED_BUT_TOKEN_NOT_JWS.useraction=If JWE tokens must be accepted, configure the mp.jwt.decrypt.key.location MicroProfile Config property or the keyManagementKeyAlias attribute in the JWT consumer configuration. Otherwise, ensure that only JWS tokens are sent to this resource.
+
+JWE_REQUIRED_BUT_TOKEN_NOT_JWE=CWWKS6064E: The token in the request is not in JSON Web Encryption (JWE) format, but only tokens that are in JWE format are accepted. If the token is in JSON Web Signature (JWS) format, remove the mp.jwt.decrypt.key.location MicroProfile Config property and the keyManagementKeyAlias attribute from the [{0}] JWT consumer configuration.
```

Comment:
similar to my suggestion for CWWKS6063E:

`The token in the request is not in JSON Web Encryption (JWE) format but the JWE decryption key location is configured. To accept tokens that are in JSON Web Signature (JWS) format, remove the mp.jwt.decrypt.key.location MicroProfile Config property and the keyManagementKeyAlias attribute from the [{0}] JWT consumer configuration.`